lib/matplotlib/colorbar.py

```diff
@@ -1474,31 +1462,20 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
             ax.set_anchor(parent_anchor)
 
     cax = fig.add_axes(pbcb, label="<colorbar>")
-
```

Comment:
This was all removed because we no longer have a special way to layout colorbars for constrained_layout, though we *do* still use `make_axes` instead of `make_axes_gridspec`